po/README.md

```diff
@@ -0,0 +1,3 @@
+### Translations
+You can translate Open Food Facts at https://translate.openfoodfacts.org. Please do not translate directly in the code, or your translations will be overwritten.
+If you have added or modified a string, you should update the crowdin-trigger branch from main/develop. This will trigger import within Crowdin, and 1 hour later, it will create a new pull request with added or modified strings in all language files, plus all added or modified translations.
```

Comment:
Wow I didn't know about this.

It's not clear to me what you mean by "update the crowdin-trigger branch from main/develop" does this mean "merging main into crowdin-trigger" ? 
Could'nt this be done by looking at po files instead of a branch (actions can be conditioned by files).

Reply:
- there are some times where you're trying to merge translations, and you don't want it auto-triggering
- we could give the file based trigger a try